app/javascript/components/LegendListElement.jsx

```diff
@@ -1,6 +1,15 @@
 import React from "react"
 import PropTypes from "prop-types"
 class LegendListElement extends React.Component {
+
+  build_search_url = (icon_name) => {
+    if (window.location.href.includes("search")){
+      return window.location.href.replace(/(search=).*?(&|$)/,'$1' + icon_name.replace(" ", "+") + '$2');
```

Comment:
I think we don't need this anymore? Since search takes into account the icon name..

Reply:
This just launches the search. It does nothing else but build out a new search based on the name of the icon. If we didnt have this, if someone had done a search and then used a quick link, it would have no route to run for the app to do the search.

Reply:
It seems cleaner (with less possibilities of regex issues) that when clicking the link of an icon that it just does crafts the search based on the icon_name.. My comment was more around wondering if we need the evaluation and the regex.